pre-processing.py

```diff
@@ -0,0 +1,228 @@
+# if kernel crashes, make sure pywin32 and pipywin32 are installed.
+# Followed instructions here: https://github.com/jupyter/notebook/issues/4909
+# import win32api
+
+import pandas as pd
+import numpy as np
+import microdf as mdf
+import os
+import us
+
+# Import data from Ipums
+person = pd.read_csv("cps_00041.csv.gz")
+# lower column names
+person.columns = person.columns.str.lower()
+# Divide by three for three years of data.
+person[["asecwt", "spmwt"]] /= 3
+
+# Create booleans for demographics
+person["adult"] = person.age >= 18
+person["child"] = person.age < 18
+
+person["black"] = person.race == 200
+person["white_non_hispanic"] = (person.race == 100) & (person.hispan == 0)
```

Comment:
Shouldn't this use the `700` threshold as below? I'd suggest flipping with the below line and then doing
```suggestion
person["white_non_hispanic"] = (person.race == 100) & ~person.hispanic
```